neurolink-demo/server.ts

```
@@ -105,8 +109,10 @@ const usageStats = {
  * Request logging middleware
  * Logs all incoming requests and updates usage statistics
  */
-const logRequest = (req, res, next) => {
-  console.log(`[${new Date().toISOString()}] ${req.method} ${req.path}`);
+const logRequest = (req: Request, _res: Response, next: NextFunction) => {
+  const safeMethod = req.method.replace(/[\r\n\x00-\x1f]/g, "");
```

Biome (2.3.14)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)